python/mxnet/kvstore/kvstore.py

```diff
@@ -67,6 +68,7 @@ def __init__(self, handle):
         self._updater = None
         self._updater_func = None
         self._str_updater_func = None
+        self._is_p3 = (os.getenv('DMLC_PS_VAN_TYPE', '') == 'p3')
```

Comment:
will u also change launch.py?

Reply:
sure. I just wanted to confirm if we are planning to only add --p3 to launch.py now and not --horovod or --byteps because of the reasons I mentioned in the previous comment

Reply:
Could you clarify a bit more on this? According to this RFC (https://github.com/apache/incubator-mxnet/issues/16795), p3 will be treated as another kvstore just like byteps/horovod. Why would it be different?

Reply:
@apeforest because the source code lives inside mxnet, it supports both device and local mode, and can share the same python KVStore class

Reply:
@apeforest We do have a separate class for P3 in the backend though.